cypress/plugins/index.js

```diff
@@ -0,0 +1,7 @@
+const { generateUser } = require('./cypress/e2e/article.cy.js');
```

Comment:
The file path in the require statement is incorrect. It should not contain 'cypress' directory at the beginning.